Testing/Unit/PowerShell/Orchestrator/Invoke-ScubaConfig.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+$OrchestratorPath = '../../../../PowerShell/ScubaGear/Modules/Orchestrator.psm1'
+$ScubaConfigPath = '../../../../PowerShell/ScubaGear/Modules/ScubaConfig/ScubaConfig.psm1'
+$TestUtilsPath = "../PsTestUtils.psm1"
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $OrchestratorPath) -Function 'Invoke-SCuBA' -Force
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ScubaConfigPath)
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $TestUtilsPath)
+
+InModuleScope Orchestrator {
+    Describe -Tag 'Orchestrator' -Name 'Invoke-Scuba with Config' {
+        BeforeAll {
+            function Remove-Resources {}
+            Mock -ModuleName Orchestrator Remove-Resources {}
+            function Import-Resources {}
+            Mock -ModuleName Orchestrator Import-Resources {}
+            function Invoke-Connection {}
+            Mock -ModuleName Orchestrator Invoke-Connection { @() }
+            function Get-TenantDetail {}
+            Mock -ModuleName Orchestrator Get-TenantDetail { '{"DisplayName": "displayName"}' }
+            function Invoke-ProviderList {}
+            Mock -ModuleName Orchestrator Invoke-ProviderList {}  
+            function Invoke-RunRego {}
+            Mock -ModuleName Orchestrator Invoke-RunRego {}
+
+            Mock -ModuleName Orchestrator Invoke-ReportCreation {}
+            function Disconnect-SCuBATenant {}
+            Mock -ModuleName Orchestrator Disconnect-SCuBATenant {}
```

Comment:
You will need to explicitly import the Connection module since it exports the Disconnect-SCuBATenant function so it can be found and mocked.

Reply:
Not sure how to do this.  The one one which this in modeled Invoke-ScubaConfig.Tests.ps1 does not do this

Reply:
Completed